core/wasmcloud-core.smithy

```diff
@@ -195,6 +200,30 @@ map HostEnvValues {
     value: String,
 }
 
+@enum([
+    {
+        value: "trace",
```

Comment:
In https://github.com/wasmCloud/weld/pull/148, you allow this to be more complex. Did you want this to be an enum?

Reply:
Yep, the host will always™️ supply one of these values. For readers, in smithy v1, enums are really "just" strings and there's no actual constraint that the value be one in this list. This enum is thus for documentation purposes